glib/src/gobject/signal_group.rs

```diff
@@ -42,6 +42,44 @@ impl SignalGroup {
         self.connect_closure(signal_name, after, RustClosure::new_local(callback));
     }
 
+    #[inline]
+    pub fn connect_notify<F>(&self, name: Option<&str>, callback: F)
+    where
+        F: Fn() + Send + Sync + 'static,
```

Comment:
You probably want to pass the object and pspec into the closure

Reply:
I'm not sure how to get the pspec of the `target` object though.

Reply:
Using https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/object/trait.ObjectExt.html#tymethod.find_property?

Reply:
It's passed as argument to the callback

Reply:
Together with the object itself